src/lib/containers/Facets.js

```diff
@@ -1,25 +1,45 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import * as SynapseConstants from 'lib/utils/SynapseConstants';
-const uuidv4 = require('uuid/v4');
-let cloneDeep = require('lodash.clonedeep');
+const cloneDeep = require("lodash.clonedeep")
+
+class CheckboxGroup extends React.Component {
+
+    render() {
+        const {element} = this.props
+        let children = []
+        let selectedFacets = this.props.selectedFacets
+        element.facetValues.forEach(
+            facetValue => {
+                children.push(
+                    <div key={facetValue.value + " " + facetValue.count}>
+                        <input defaultChecked={facetValue.isSelected} onClick={this.props.clickHandler({selectedFacets: selectedFacets, value: facetValue.value, columnName: element.columnName})} id={element.columnName + " " + facetValue.value + " " + facetValue.count} type="checkbox"/>
```

Comment:
does the id need to be set to this?
element.columnName + " " + facetValue.value + " " + facetValue.count
if required, I think it precludes the possibility of having multiple tables being rendered on the same page (at least, it would be a violation of the html spec to have multiple elements with the same id).